src/pages/Swap/index.tsx

```diff
@@ -463,6 +466,7 @@ export default function Swap() {
                 </Text>
               </ButtonError>
             )}
+            {showApproveFlow && <ProgresSteps steps={[approval === ApprovalState.APPROVED]} />}
```

Comment:
typo in progress, also only one step?

Reply:
One step because on swap completion it resets - i just made the component handle arbitrary amount of steps, so could use it in the future for multistep